self-hosted/install/installation-docker.md

```diff
@@ -1,83 +1,82 @@
+instance on any local system, from a pre-built container. 
 
 < Skip/>
 
+This section shows you how to:
 
-This is the simplest method to self-host TimescaleDB, and it means you
-always have access to the latest version without worrying about local
-dependencies. You can access the Docker image directly from locally installed
-PostgreSQL client tools such as `psql`.
-
+* [Install and configure TimescaleDB on PostgreSQL](#install-and-configure-timescaledb-on-postgresql) - set up
+  a self-hosted PostgreSQL instance to efficiently run TimescaleDB1.
+* [Add the TimescaleDB extension to your database](#add-the-timescaledb-extension-to-your-database) - enable TimescaleDB features and
+  performance improvements on a database.
 
 <Highlight type="warning">
+
 If you have already installed PostgreSQL using a method other than the pre-built
-container provided here, you could encounter errors following these
-instructions. It is safest to remove any existing PostgreSQL installations
-before you begin. If you want to keep your current PostgreSQL installation, do
-not install TimescaleDB using this method.
-[Install from source](/self-hosted/latest/install/installation-source/)
-instead.
+container provided here, you may encounter errors
+following these install instructions. Best practice is to full remove any existing PostgreSQL
+installations before you begin.
+
+To keep your current PostgreSQL installation, [Install from source][install-from-source].
 </Highlight>
 
-<Procedure>
+### Prerequisites
 
-## Installing TimescaleDB from a Docker container
+To connect to a PostgreSQL installation on Docker, you need to install:
 
-1.  Install Docker, if you don't already have it. For packages and
-    instructions, see the [Docker installation documentation][docker-install].
-1.  At the command prompt, run the TimescaleDB Docker image:
+- [Docker][docker-install]
+- [psql][install-psql]
 
-    ```bash
-    docker pull timescale/timescaledb-ha:pg16
-    ```
 
-<Highlight type="important">
-The [`timescaledb-ha`](https://hub.docker.com/r/timescale/timescaledb-ha) image
-offers the most complete TimescaleDB experience. It
-includes the
-[TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit),
-and support for PostGIS and Patroni. If you need the smallest possible image, use
-the `timescale/timescaledb:latest-pg14` image instead.
-</Highlight>
+## Install and configure TimescaleDB on PostgreSQL
 
-</Procedure>
+Best practice for self-hosting TimescaleDB is to install the latest version of PostgreSQL and
+TimescaleDB on a [supported platform](#supported-platforms) using the packages supplied by Timescale.
+
+<SelfHostedDocker />
+
+
+## Add the TimescaleDB extension to your database
+
+For improved performance, you enable TimescaleDB on each database on your self-hosted PostgreSQL instance.
+This section shows you how to enable TimescaleDB for a new database in PostgreSQL using `psql` from the command line.
```

Comment:
The timescaledb extension is pre-created in the default `postgres` database in both the -ha and non-ha docker images. It also gets created by default in new databases in these images.

<img width="1071" alt="image" src="https://github.com/timescale/docs/assets/4974668/f8283ee7-9a60-40fc-960c-4ed169b8deb6">

<img width="999" alt="image" src="https://github.com/timescale/docs/assets/4974668/351168db-e813-4136-9b88-c3bd91c831a3">